content/correspondence/getting-started/developer-guides/serviceowner/_index.nb.md

```
@@ -43,15 +47,18 @@ En bruker med denne tilgangen kan deretter delegere tilgangen til virksomhetsbru
-- `altinn:correspondence.write` 
 
-For å få tilgang til scopes må du sende en forespørsel til: servicedesk@altinn.no 
-Forespørselen må inneholde de scopes du trenger. Vær obs på at du kan trenge flere scopes for integrasjonen din enn bare altinn:correspondence.write. 
-Utfyllende liste over scopes finner du her: 
-https://docs.altinn.studio/nb/api/authentication/digdirscopes/ 
+- `altinn:correspondence.write`
+
+For å få tilgang til scopes må du sende en forespørsel til: servicedesk@altinn.no
+Forespørselen må inneholde de scopes du trenger. Vær obs på at du kan trenge flere scopes for integrasjonen din enn bare altinn:correspondence.write.
+Utfyllende liste over scopes finner du her:
+https://docs.altinn.studio/nb/api/authentication/digdirscopes/
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)